tests/tests/abi_tests.cpp

```diff
@@ -390,15 +390,32 @@ struct abi_gen_helper {
   bool generate_abi(const char* source, const char* abi, bool opt_sfs=false) {
 
     std::string include_param = std::string("-I") + eosiolib_path;
+    std::string pfr_include_param = std::string("-I") + pfr_include_path;
     std::string boost_include_param = std::string("-I") + boost_include_path;
     std::string stdcpp_include_param = std::string("-I") + eosiolib_path + "/libc++/upstream/include";
     std::string stdc_include_param = std::string("-I") + eosiolib_path +  "/musl/upstream/include";
 
-     abi_def output;
-    bool res = runToolOnCodeWithArgs(new generate_abi_action(false, opt_sfs, "", output), source,
-      {"-fparse-all-comments", "--std=c++14", "--target=wasm32", "-ffreestanding", "-nostdlib", "-nostdlibinc", "-fno-threadsafe-statics", "-fno-rtti",  "-fno-exceptions", include_param, boost_include_param, stdcpp_include_param, stdc_include_param });
+    abi_def output;
 
+    std::string contract;
+    std::vector<std::string> actions;
+
+    bool res = runToolOnCodeWithArgs(new find_eosio_abi_macro_action(contract, actions, ""), source,
+      {"-fparse-all-comments", "--std=c++14", "--target=wasm32", "-ffreestanding", "-nostdlib",
+      "-nostdlibinc", "-fno-threadsafe-statics", "-fno-rtti",  "-fno-exceptions",
+      include_param, boost_include_param, stdcpp_include_param,
+      stdc_include_param, pfr_include_param }
+    );
+    FC_ASSERT(res == true);
+
+    res = runToolOnCodeWithArgs(new generate_abi_action(false, opt_sfs, "", output, contract, actions), source,
+      {"-fparse-all-comments", "--std=c++14", "--target=wasm32", "-ffreestanding", "-nostdlib",
+      "-nostdlibinc", "-fno-threadsafe-statics", "-fno-rtti",  "-fno-exceptions",
+      include_param, boost_include_param, stdcpp_include_param,
+      stdc_include_param, pfr_include_param }
```

Comment:
Since this vector of args is the same here and above, I would create the vector above this and pass it in so that it is easier to maintain.